man/use_cassette.Rd

```diff
@@ -134,28 +134,6 @@ block.
 \code{use_cassette()} block starts on.
 }
 
-\section{Cassette names}{
```

Comment:
is it intended that use_cassette loses the __Cassette names__ section?

Reply:
See my first comment, I can't avoid this